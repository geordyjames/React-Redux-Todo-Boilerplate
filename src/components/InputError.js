import React from 'react';

const InputError = ({error}) => (
  <div className="uk-alert-danger" data-uk-alert>
    <p> {error} </p>
  </div>
);

export default InputError;