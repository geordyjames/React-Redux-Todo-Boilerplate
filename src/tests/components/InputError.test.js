import React from 'react';
import { shallow } from 'enzyme';
import InputError from '../../components/InputError';

test('should render InputError with error prop correctly', () => {
  const wrapper = shallow(<InputError error="Some Error Message" />);
  expect(wrapper).toMatchSnapshot();
});

test('should render InputError with no error prop correctly', () => {
  const wrapper = shallow(<InputError />);
  expect(wrapper).toMatchSnapshot();
});