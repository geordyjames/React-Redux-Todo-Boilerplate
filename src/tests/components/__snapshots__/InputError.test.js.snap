// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render InputError with error prop correctly 1`] = `
<div
  className="uk-alert-danger"
  data-uk-alert={true}
>
  <p>
     
    Some Error Message
     
  </p>
</div>
`;

exports[`should render InputError with no error prop correctly 1`] = `
<div
  className="uk-alert-danger"
  data-uk-alert={true}
>
  <p>
     
     
  </p>
</div>
`;
